core/conversion/conversion.cpp

```diff
@@ -337,12 +369,28 @@ void ConvertBlockToNetDef(
     } else if (to_eval) {
       auto eval = EvaluateNode(ctx, n);
       if (eval) {
-        if (!eval.value().isTensor()) {
+        if (n->outputs().size() > 1) { // For ListUnpack scenario
+          if (eval.value().isList()) {
+            LOG_DEBUG(
+                ctx->logger,
+                "Found the evaluated value to be a list" << eval.value() << " for node: " << util::node_info(n));
+            auto eval_list = eval.value().toList();
+            TRTORCH_CHECK(
+                eval_list.size() == n->outputs().size(),
+                "Size of evaluated results: " << eval_list.size() << " and node outputs size: " << n->outputs().size()
+                                              << " must match.");
+            for (int i = 0; i < eval_list.size(); i++) {
+              auto eval_output = eval_list.get(i);
+              ctx->AssociateValueAndIValue(n->output(i), eval_output);
```

Comment:
We should move the debug statement above to here and list out all found results.

Reply:
Done